core/src/main/java/com/yahoo/oak/FirstLevelHashArray.java

```diff
@@ -106,24 +106,43 @@ private int calculateChunkSize(int inputLsbForSecondLevel) {
         return (int) Math.ceil(Math.pow(2, lsbForSecondLevel));
     }
 
-    private int calculateKeyHash(K key, ThreadContext ctx) {
```

Comment:
it was removed because because it was not used, and it seemed to me very misleading to have a function named "calculateKeyHash" that do not calculate key hash